package-lock.json

```diff
@@ -11200,16 +11200,51 @@
 			},
 			"dependencies": {
 				"chalk": {
-					"version": "2.4.2",
-					"resolved": "https://registry.npmjs.org/chalk/-/chalk-2.4.2.tgz",
-					"integrity": "sha512-Mti+f9lpJNcwF4tWV8/OrTTtF1gZi+f8FqlyAdouralcFWFQWF2+NgCHShjkCb+IFBLq9buZwE1xckQU4peSuQ==",
+					"version": "4.0.0",
```

Comment:
Huh, this lock file handling is really annoying. Now it still keeps the local copy for each WordPress package 😕 

I guess the only way to fix it is by doing surgery 😄 I usually remove such `dependencies` section manually and it no longers gets added there if the same dependency is already listed in top level.

Reply:
I think I was successful in doing this in 76b6501e78 . Strange that it's something to do manually. Seems to work as expected though (at least no changes after running `npm install` once or twice more, with or without `node_modules` already present).